src/main/java/com/epam/training/auction_backend/controllers/SessionProvider.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.training.auction_backend.controllers;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+
+import com.epam.training.auction_backend.entity.Auction;
+import com.epam.training.auction_backend.entity.User;
+
+public class SessionProvider {
```

Comment:
This class is not needed if you use CrudRepository interface as mentioned before.